File: Semestrul 4/MPP2/Problema8SC/Problema8SC/Problema8Services/src/main/java/problema8/services/ProbExceptions.java
package problema8.services;

public class ProbExceptions extends RuntimeException {
    public ProbExceptions() {
    }
    public ProbExceptions(String message) {
        super(message);
    }
    public ProbExceptions(String message, Throwable cause) {
        super(message, cause);
    }
}

File: Semestrul 4/MPP2/Client-Server_2Limbaje/Problema8SC_CSharp/Services/ProbExceptions.cs
namespace Problema8SC_CSharp.Services;

public class ProbExceptions : Exception
{
    public ProbExceptions(): base() { }
    public ProbExceptions(string message) : base(message) { }
    public ProbExceptions(string message, Exception innerException) : base(message, innerException) { }
}